src/h5web/vis-packs/core/line/LineVis.tsx

```diff
@@ -104,15 +104,15 @@ function LineVis(props: Props) {
           ordinates={dataArray.data as number[]}
           errors={errorsArray && (errorsArray.data as number[])}
           showErrors={showErrors}
-          color={curveColor}
+          color={curveColor || 'midnightblue'}
```

Comment:
I cannot provide the default values in CSS, because these colours are not assigned to actual CSS properties. If I define `--h5w-line--curveColor` in `LineVis.module.css`, users won't be able to override it from further up the DOM tree.

For all the other variables, I'll be able to do something like `stroke: var(--h5web-axis--tickColor, mydefaultcolor)` in CSS directly.